src/screenComponents/databaseView.cpp

```diff
@@ -33,7 +33,7 @@ DatabaseViewComponent::DatabaseViewComponent(GuiContainer* owner)
         }
         display(entry);
     });
-    item_list->setPosition(0, 0, ATopLeft)->setMargins(20, 20, 20, 130)->setSize(400, GuiElement::GuiSizeMax);
```

Comment:
This extra room is for the switch back to science button.

Reply:
You are right: The switch button on the science screen was the thing I forgot.